tests/integration/utils2.go

```diff
@@ -1906,6 +1944,22 @@ func assertRequestResults(
 	return false
 }
 
+func convertToArrayOfMaps(t testing.TB, value any) []map[string]any {
```

Comment:
question: these (and others) are changes from my PR that was merged. How did it end up here?

Reply:
I think I copied these on accident when I was mid-refactor and didn't realize my local branch was behind. When I merge the develop branch these should appear as unchanged.